src/Integration/Binding/SolutionBindingOperation.cs

```diff
@@ -210,14 +192,25 @@ public void Prepare(CancellationToken token)
                 Debug.Assert(sourceControlledFileSystem.FileExistOrQueuedToBeWritten(info.FilePath), "Expected a rule set to pend pended");
             }
 
-            foreach (var binder in this.childBinder)
+            foreach (var project in projects)
             {
                 if (token.IsCancellationRequested)
                 {
                     return;
                 }
-
-                binder.Prepare(token);
+                if (projectBinderFactory.Get(project) is RoslynProjectBinder)
+                {
+                    var languageForProject = ProjectToLanguageMapper.GetLanguageForProject(project);
+                    var bindingConfigFile = GetBindingConfig(languageForProject) as IBindingConfigFileWithRuleset;
+                    var binder = new ProjectBindingOperation(serviceProvider, project, bindingConfigFile);
+                    binder.Initialize();
```

Comment:
I need to unite now the calls to Initialize and Prepare, since the bindingConfigFile is not available in the solution Initialize phase, only on Prepare.